src/libraries/System.Net.Http/src/System/Net/Http/DelegatingHandler.cs

```diff
@@ -51,7 +53,7 @@ protected internal override Task<HttpResponseMessage> SendAsync(HttpRequestMessa
                 throw new ArgumentNullException(nameof(request), SR.net_http_handler_norequest);
             }
             SetOperationStarted();
-            return _innerHandler.SendAsync(request, cancellationToken);
+            return _innerHandler!.SendAsync(request, cancellationToken);
```

Comment:
Why are we sure that it is not null here?

Reply:
`SetOperationStarted();` method above checks it for null and throws